os/kernel/binary_manager/binary_manager.h

```diff
@@ -251,16 +272,15 @@ void binary_manager_get_info_with_name(int request_pid, char *bin_name);
 void binary_manager_get_info_all(int request_pid);
 void binary_manager_get_state_with_name(int request_pid, char *bin_name);
 void binary_manager_send_response(char *q_name, void *response_msg, int msg_size);
-int binary_manager_register_ubin(char *name, uint32_t version, uint8_t load_priority);
-void binary_manager_scan_ubin_all(void);
-int binary_manager_scan_ubin(int bin_idx);
-int binary_manager_read_header(char *path, user_binary_header_t *header_data, bool crc_check);
+int binary_manager_read_header(int type, char *devpath, void *header_data, bool crc_check);
```

Comment:
Most of the binary manager functions use `BINMGR_OK`, but this function uses `OK`.
Please unify them.

Reply:
I updated the return value of `binary_manager_read_header` to `BINMGR_OK`